src/rust/iced-x86/src/instruction.rs

```diff
@@ -2307,15 +2307,14 @@ impl Instruction {
 
 	/// Gets the virtual address of a memory operand
 	///
-	/// # Panics
-	///
-	/// Panics if `operand` is invalid
+	/// Returns `None` if `operand` is invalid.
```

Comment:
This is incorrect, it will still panic if `operand` is invalid since it calls `op_kind()`. The comment should be restored.

Reply:
Good catch, thanks!